### PR TITLE
chore(l10n): add initial ftl from new settings

### PR DESCRIPTION
**Pull request by chenba**
This commit add the initial set of strings from FxA's new settings frontend.

### CODE REVIEW

File: locale/templates/settings.ftl [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+se-verify-session-error = Sorry, there was a problem verifying your session.
+# Button to remove the secondary email
+se-remove-email =
+  .title = Remove email
+# Button to refresh secondary email status
+se-refresh-email =
+  .title = Refresh email
+se-unverified = unverified
+se-resend-code = Verification needed. <button>Resend verification code</button>
+  if it's not in your email or spam.
```
Comment (peiying2) [inline locale/templates/settings.ftl:351]: `your email or spam` probably needs to be reworded. Maybe `your inbox or spam folder`?

**Review by peiying2 — CHANGES_REQUESTED** [review]
Please check the initial review comments. I will go over the .ftl syntax and comment if necessary.

File: locale/templates/settings.ftl [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+nav-security = Security
+nav-connected-services = Connected Services
+nav-paid-subs = Paid Subscriptions
+nav-email-comm = Email Communications
+
+## Two Step Authentication - replace recovery code
+
+tfa-replace-code-error = There was a problem replacing your recovery codes.
+tfa-replace-code-success = New codes have been created. Save these one-time use
+  codes in a safe place—you’ll need them to access your account if you don’t
```
Comment (peiying2) [inline locale/templates/settings.ftl:122]: Add space before and after `—` em dash.

File: locale/templates/settings.ftl [hunk 3]
```diff
@@ -0,0 +1,399 @@
...
+# Button to refresh secondary email status
+se-refresh-email =
+  .title = Refresh email
+se-unverified = unverified
+se-resend-code = Verification needed. <button>Resend verification code</button>
+  if it's not in your email or spam.
+# Button to make secondary email the primary
+se-make-primary = Make primary
+se-default-content = Access your account if you can't log in to your primary email.
+se-content-note = Note: a secondary email won't restore your information—you'll
```
Comment (peiying2) [inline locale/templates/settings.ftl:355]: Add space before and after `—` em dash

File: locale/templates/settings.ftl [hunk 4]
```diff
@@ -0,0 +1,399 @@
...
+  or <br /><linkExternal>send a download link to your device.</linkExternal>
+
+## Connected services section
+
+cs-heading = Connected Services
+cs-description = Everything you are using and signed into.
+cs-cannot-refresh = Sorry, there was a problem refreshing the list of connected
+  services.
+cs-cannot-disconnect = Client not found, unable to disconnect
+cs-logged-out = Logged out of { $service }.
```
Comment (peiying2) [inline locale/templates/settings.ftl:50]: There needs to be a reference of the variable at the beginning of this block, as seen in the example here: https://firefox-source-docs.mozilla.org/l10n/fluent/review.html#comments

**Comment by chenba** [comment]
@peiying2 Thank you so much for the feedback.  I'll mark each comment as resolved as I fix them in the FxA repo.

File: locale/templates/settings.ftl [hunk 5]
```diff
@@ -0,0 +1,399 @@
...
+delete-account-header =
+ .title = Delete Account
+
+delete-account-step-1-2 = Step 1 of 2
+delete-account-step-2-2 = Step 2 of 2
+
+delete-account-confirm-title = You've connected your { -firefox-account } to { -mozilla } products that keep you secure and productive on the web:
+
+# Note: don't translate this product name.
+mozilla-vpn = Mozilla VPN
```
[the remark this answers is not in the document]
Comment (chenba) [reply locale/templates/settings.ftl:178]: I can move them.

They are used in the delete account steps.  It looks like this:
image

[the remark this answers is not in the document]
Comment (peiying2) [reply locale/templates/settings.ftl:178]: Ideally, you have a separate file called "brands.ftl" so you can keep adding there if need be and can update the list (addition or removal). Or you put them all at the top of the file, and reference the name later in the file.

File: locale/templates/settings.ftl [hunk 6]
```diff
@@ -0,0 +1,399 @@
...
+firefox-lockwise = Firefox Lockwise
+# Note: don't translate this product name.
+firefox-monitor = Firefox Monitor
+
+delete-account-acknowledge = Please acknowledge that by deleting your account:
+
+delete-account-chk-box-1 =
+ .label = Any paid subscriptions you have will be cancelled
+delete-account-chk-box-2 =
+ .label = Any may lose saved information and features within { -mozilla } products
```
[the remark this answers is not in the document]
Comment (chenba) [reply locale/templates/settings.ftl:189]: Appears to be a typo.  It should've been 'You may lose...".

**Comment by chenba** [comment]
I pushed another commit with the changes based on the feedback for easier re-review.  I can squash into a single commit before merging.  Thanks!

File: locale/templates/settings.ftl (hunk 5 above)
[the remark this answers is not in the document]
Comment (chenba) [reply locale/templates/settings.ftl]: Sorry, I didn't see this message prior to pushing my latest commit.

For clarification, is having the "brands.ftl" file for our (FxA engr) benefit, or the localizers?  This settings.ftl is actually generated from smaller ftl files in the FxA repo.  But if it helps localizers to have a "brands.ftl", we definitely can do that

[the remark this answers is not in the document]
Comment (peiying2) [reply locale/templates/settings.ftl]: It's good practice to have a separate one so it's easy to address brand related terms. Technically, brands are left untranslated, product and feature names are not translated for the most part with a few exception (Accounts in FxA, Sync). This file also allows it to grow in the future when we have more products to add. It's also easy to reference to if a product name is not coded as a placeable. Firefox and Mozilla.org have a brands.ftl each. You can keep the PR as is for now. Wil and l10n team are going to have a meeting on Monday regarding how to manage this release.

File: locale/templates/settings.ftl [hunk 7]
```diff
@@ -0,0 +1,407 @@
...
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Globally reused strings
+
+# Note: the company name, Mozilla, is not translated.
+-mozilla = Mozilla
+-firefox = Firefox
+-firefox-accounts = Firefox Accounts
+-firefox-account = Firefox account
+-sync = Sync
```
Comment (peiying2) [inline locale/templates/settings.ftl:12]: Apologize for asking you to revert back to your original setup. Marketing's suggestion is to leave this feature name out.

File: locale/templates/settings.ftl [hunk 8]
```diff
@@ -0,0 +1,407 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Globally reused strings
+
+# Note: the company name, Mozilla, is not translated.
```
Comment (peiying2) [inline locale/templates/settings.ftl:7]: If we call out the names that shouldn't be translated, let's include Firefox as well.

**Review by peiying2 — CHANGES_REQUESTED** [review]
I reverted the comment on Sync after checking with team. I will get back to you on any comment on the technical Fluent related review by tomorrow morning.

FYI, had a meeting with Wil on what to do with the PR. The suggested changes are still valid, but we may not roll out to l10n.

File: locale/templates/settings.ftl [hunk 9]
```diff
@@ -0,0 +1,407 @@
...
+cs-disconnect-advice-confirm = Okay, got it
+cs-disconnect-lost-advice-heading = Lost or stolen device disconnected
+cs-disconnect-lost-advice-content = Since your device was lost or stolen, to
+  keep your information safe, you should change your { -firefox-account } password
+  in your account settings. You should also look for information from your
+  device manufacturer about erasing your data remotely.
+cs-disconnect-suspicious-advice-heading = Suspicious device disconnected
+cs-disconnect-suspicious-advice-content = If the disconnected device is indeed
+  suspicious, to keep your information safe, you should change your { -firefox }
+  account password in your account settings. You should also change any other
```
Comment (peiying2) [inline locale/templates/settings.ftl:98]: Include `account` in { -firefox-account }

**Review by peiying2 — COMMENTED** [review]
@flodolo please go review the PR. Additional questions I have:

- Usages of ## and # 
- Line space after ## (there is one line space) or # (no line space)
- Section break - without the ## or #, should there be spaces between section.  
- Which one, # or ## would a localizer see in Pontoon if immediately followed by IDed strings. The same comment is shown in every string within a block (no line space)?

File: locale/templates/settings.ftl [hunk 10]
```diff
@@ -0,0 +1,407 @@
...
+# Verify secondary email page
+
+verify-secondary-email-error = There was a problem sending the verification code.
+verify-secondary-email-page-title =
+  .title = Secondary email
+verify-secondary-email-verification-code =
+  .label = Enter your verification code
+verify-secondary-email-cancel-button = { fxa-cancel-button }
+verify-secondary-email-verify-button = Verify
+# Note: include the placeholder '{ $email }' without translation. It will be replaced with the user's email address.
```
Comment (flodolo) [inline locale/templates/settings.ftl:253]: I'd replace this comment with just the standard format for the variable.

**Review by flodolo — CHANGES_REQUESTED** [review]

File: locale/templates/settings.ftl [hunk 11]
```diff
@@ -0,0 +1,407 @@
...
+
+delete-account-step-1-2 = Step 1 of 2
+delete-account-step-2-2 = Step 2 of 2
+
+delete-account-confirm-title = You've connected your { -firefox-account } to { -mozilla } products that keep you secure and productive on the web:
+
+delete-account-acknowledge = Please acknowledge that by deleting your account:
+
+delete-account-chk-box-1 =
+ .label = Any paid subscriptions you have will be cancelled
```
[the remark this answers is not in the document]
Comment (chenba) [reply locale/templates/settings.ftl:195]: TIL `cancelled` is Canadian!  (Which was why it always looked weird to me.)

**Comment by chenba** [comment]
Thanks for the great feedback, @flodolo.  I'll mark each comment resolved as I make the suggested change in the FxA repo.

File: locale/templates/settings.ftl [hunk 12]
```diff
@@ -0,0 +1,407 @@
...
+avatar-your-avatar =
+  .alt = Your avatar
+avatar-default-avatar =
+  .alt = Default avatar
+
+## Connect another device promo
+
+connect-another-fx-mobile = Get { -firefox } on mobile or tablet
+connect-another-find-fx-mobile = Find { -firefox } in the { -google-play } and { -app-store }
+  or <br /><linkExternal>send a download link to your device.</linkExternal>
```
[the remark this answers is not in the document]
Comment (chenba) [reply locale/templates/settings.ftl:52]: I think the expectation was that Fluent can auto handle the indentation correctly (per https://www.projectfluent.org/fluent/guide/multiline.html), but that was an awkward spot to have a line break.  I'll update it so it's all one line.

[the remark this answers is not in the document]
Comment (chenba) [reply locale/templates/settings.ftl:52]: Oh I think I misunderstood.  You mean the space that precedes the br tag?  I don't think it'll be a problem.

File: locale/templates/settings.ftl [hunk 13]
```diff
@@ -0,0 +1,407 @@
...
+  .title = Refresh connected services
+
+# Link text to a support page on missing or duplicate devices
+cs-missing-device-help = Missing or duplicate items?
+
+cs-disconnect-sync-heading = Disconnect from { -sync }
+# $device is the name of the device being disconnected
+cs-disconnect-sync-content = Your browsing data will remain on your
+  device ({ $device }), but it will no longer sync with your account.
+cs-disconnect-sync-reason = What's the main reason for disconnecting this
```
[the remark this answers is not in the document]
Comment (chenba) [reply locale/templates/settings.ftl:77]: I'm certainly not aware of such rules.  (I've been on the FxA team for a little over 1.5 years.)  I'll update the `'` and `""`  to `’` and `“”`.

File: locale/templates/settings.ftl [hunk 14]
```diff
@@ -0,0 +1,407 @@
...
+tfa-replace-code-error = There was a problem replacing your recovery codes.
+tfa-replace-code-success = New codes have been created. Save these one-time use
+  codes in a safe place — you’ll need them to access your account if you don’t
+  have your mobile device.
+
+# Avatar change page
+
+avatar-page-title =
+  .title = Avatar
+avatar-page-add-photo = Add Photo
```
[the remark this answers is not in the document]
Comment (6a68) [reply locale/templates/settings.ftl:144]: (I'm taking over the remaining items here, marking comments as resolved as they are addressed in the branch.) These strings are only used in two spots each; the variable is used to minimize the amount of duplicated work we ask of the localizers. I'll update the remove-photo string to use a variable for the title attribute as well.

**Comment by peiying2** [comment]
Thanks to flod for all the review comments (I am taking notes too for future reviews). I am not clear what the team's plan is for this PR. From the discussion we had with Wil, if the new Setting were to go live per the original launch plan, and migration of existing translated strings from .po to .ftl is not doable, then this PR needs to be configured so the new strings won't land in Pontoon.

**Comment by 6a68** [comment]
> I am not clear what the team's plan is for this PR

Hey @peiying2, Barry is working on another project this week, so I'm taking over this work. We are planning to migrate the existing translations from the .po files to the .ftl files (this is tracked by https://github.com/mozilla/fxa/issues/7459). I'm also going to land fixes in FxA based on the review comments here (in https://github.com/mozilla/fxa/pull/7540).

We can plan to close this for now, and I can open a new PR that includes the migrated existing strings, once I've pulled them over using the new format. Does that sound good? If so, feel free to just close this PR.

**Comment by 6a68** [comment]
Update: we're going to just land this file as-is. It turns out there aren't many reused strings. Reopening.

**Comment by 6a68** [comment]
@flodolo Hi, I can't seem to re-request a review from you using the github interface, but this PR has been updated, if you have bandwidth to take another look.

File: locale/templates/settings.ftl [hunk 15]
```diff
@@ -2,54 +2,64 @@
...
+##
+## Declension should be avoided where possible, leaving the original
+## brand unaltered in prominent UI positions.
+##
+## For further details, consult:
+## https://mozilla-l10n.github.io/styleguides/mozilla_general/#brands-copyright-and-trademark
+
+-brand-mozilla = Mozilla
+-brand-firefox = Firefox
+-product-firefox-accounts = Firefox Accounts
```
Comment (peiying2) [inline locale/templates/settings.ftl:21]: Both `Accounts` and `Account` are localizable.

File: locale/templates/settings.ftl [hunk 16]
```diff
@@ -70,36 +80,54 @@ cs-refresh-button =
...
 cs-disconnect-sync-opt-duplicate = Duplicate
 cs-disconnect-sync-opt-not-say = Rather not say
 
+##
+
 cs-disconnect-advice-confirm = Okay, got it
 cs-disconnect-lost-advice-heading = Lost or stolen device disconnected
 cs-disconnect-lost-advice-content = Since your device was lost or stolen, to
-  keep your information safe, you should change your { -firefox-account } password
+  keep your information safe, you should change your { -product-firefox-account } password
```
Comment (peiying2) [inline locale/templates/settings.ftl:109]: should be plural `{ -product-firefox-accounts }`

File: locale/templates/settings.ftl [hunk 17]
```diff
@@ -180,78 +214,91 @@ pw-change-new-password =
...
   .label = Enter your verification code
-verify-secondary-email-cancel-button = { fxa-cancel-button }
+verify-secondary-email-cancel-button = Cancel
 verify-secondary-email-verify-button = Verify
-# Note: include the placeholder '{ $email }' without translation. It will be replaced with the user's email address.
-verify-secondary-email-please-enter-code = Please enter the verification code that was sent to { $email } within 5 minutes.
+# This string is instruction in a form.
+# Variables:
+#   $email (String) - the user's email address, which does not need translation.
+verify-secondary-email-please-enter-code = Please enter the verification code that was sent to <strong>{$email}</strong> within 5 minutes.
```
Comment (peiying2) [inline locale/templates/settings.ftl:299]: space before and after variable? `<strong>{ $email }</strong>`

File: locale/templates/settings.ftl [hunk 18]
```diff
@@ -180,78 +214,91 @@ pw-change-new-password =
...
 verify-secondary-email-page-title =
   .title = Secondary email
 verify-secondary-email-verification-code =
   .label = Enter your verification code
-verify-secondary-email-cancel-button = { fxa-cancel-button }
+verify-secondary-email-cancel-button = Cancel
 verify-secondary-email-verify-button = Verify
-# Note: include the placeholder '{ $email }' without translation. It will be replaced with the user's email address.
-verify-secondary-email-please-enter-code = Please enter the verification code that was sent to { $email } within 5 minutes.
+# This string is instruction in a form.
```
Comment (peiying2) [inline locale/templates/settings.ftl:296]: Add `an` in `This string is an instruction...`

File: locale/templates/settings.ftl [hunk 19]
```diff
@@ -0,0 +1,471 @@
...
+
+## Delete account page
+
+delete-account-header =
+ .title = Delete Account
+
+delete-account-step-1-2 = Step 1 of 2
+delete-account-step-2-2 = Step 2 of 2
+
+delete-account-confirm-title = You’ve connected your { -product-firefox-account } to { -brand-mozilla } products that keep you secure and productive on the web:
```
Comment (peiying2) [inline locale/templates/settings.ftl:227]: Plural accounts `{ -product-firefox-accounts }`

**Review by peiying2 — CHANGES_REQUESTED** [review]
Thanks for the fix. A few more missed.

Flod is off on Friday.

**Comment by clouserw** [comment]
@peiying2 Jared pushed an updated patch for review

**Review by peiying2 — APPROVED** [review]
Looks good to me. If we have any issues, I will report back to get it fixed.

**Pull request merged**